src/tools/cephfs/top/cephfs-top

```diff
@@ -786,6 +796,20 @@ class FSTop(FSTopBase):
                         wrap(client_meta[CLIENT_METADATA_MOUNT_ROOT_KEY], hlen), curses.A_DIM)
                 else:
                     self.fsstats.addstr(y_coord, xp, "N/A", curses.A_DIM)
+            elif item == FS_TOP_MAIN_WINDOW_COL_MNTPT_TYPE:
+                if "kernel_version" in client_meta:
+                    hlen = len(item) + ITEMS_PAD_LEN
+                    self.fsstats.addstr(
+                        y_coord, xp, wrap("kernel", hlen), curses.A_DIM)
+                elif CLIENT_METADATA_MOUNT_POINT_KEY in client_meta \
+                        and client_meta[CLIENT_METADATA_MOUNT_POINT_KEY] != 'N/A':
+                    hlen = len(item) + ITEMS_PAD_LEN
+                    self.fsstats.addstr(
+                        y_coord, xp, wrap("fuse", hlen), curses.A_DIM)
+                else:
+                    hlen = len(item) + ITEMS_PAD_LEN
+                    self.fsstats.addstr(
+                        y_coord, xp, wrap("libcephfs", hlen), curses.A_DIM)
```

Comment:
Does this mean we don't have a single/unique metric, which could be checked to determine the mount_type?

Reply:
While giving the number of clients, we were just subtracting the number of kernel and fuse from the total to give this number. I used that same approach to show the mount_type too.